### DISCUSSION TITLE
Update Ideas

### DISCUSSION THREAD

**Discussion by alphxn** [Vatsal-wire/Discord-multipurpose-bot#25] [General]
Hey, awesome project you own here! I would like to give you guys some ideas and how to could be made functional again. I'm not a good developer so might say some things wrong don't mind them. First off, I would recommend is to use nextcord.py, a much better fork of the discord.py handler (discord.py without updates). It's very easy to use it. just change "discord" to "nextcord" that is the most important thing, then comes slash command syntax. That could be tricky but it is worth it in my opinion. You could also update the readme page a lot, like, add ways you can customize it, of course how to fork it and host it on local host or whatever. I don't have much more to tell you guys to add, just update the code please thank you. 
- Alphon

**Comment by EmmmaTech** [comment]
> Hey, awesome project you own here! I would like to give you guys some ideas and how to could be made functional again. I'm not a good developer so might say some things wrong don't mind them. First off, I would recommend is to use nextcord.py, a much better fork of the discord.py handler (discord.py without updates). It's very easy to use it. just change "discord" to "nextcord" that is the most important thing, then comes slash command syntax. That could be tricky but it is worth it in my opinion. You could also update the readme page a lot, like, add ways you can customize it, of course how to fork it and host it on local host or whatever. I don't have much more to tell you guys to add, just update the code please thank you. 
> 
> - Alphon
> 
>  

Honestly their preference of discord.py fork is up to them. Stop trying to convince them

**Comment by ImNimboss** [comment]
> Hey, awesome project you own here! I would like to give you guys some ideas and how to could be made functional again. I'm not a good developer so might say some things wrong don't mind them. First off, I would recommend is to use nextcord.py, a much better fork of the discord.py handler (discord.py without updates). It's very easy to use it. just change "discord" to "nextcord" that is the most important thing, then comes slash command syntax. That could be tricky but it is worth it in my opinion. You could also update the readme page a lot, like, add ways you can customize it, of course how to fork it and host it on local host or whatever. I don't have much more to tell you guys to add, just update the code please thank you.
> 
> * Alphon

@alphxn Hi! I hope you realise that there are a wide variety of forks to choose from and nextcord is not the only one? Everyone has their **own different opinions** on what discord.py fork to use and **every fork has its own advantages.** So please, kindly do not go around preaching your favorite fork :) This applies whether it be Discord, GitHub or anywhere else.

Although if you gave valid reasons for switching to nextcord over Pycord maybe your opinion would actually be considered.

**Comment by pogrammar** [comment]
>  You could also update the readme page a lot, like, add ways you can customize it, of course how to fork it and host it on local host or whatever. I don't have much more to tell you guys to add, just update the code please thank you.
> 
> * Alphon

@alphxn I will update the readme page! thank you

**Comment by pogrammar** [comment]
and @alphxn your own repo is coded in pycord bruh ?
![image](https://user-images.githubusercontent.com/97349629/150623929-c38106fd-d1d8-4104-a4be-f905c64dc7a3.png)

**Reply by pogrammar** [reply]
Like bruh why..?

**Comment by ImNimboss** [comment]
lmfaoooo

**Comment by EmmmaTech** [comment]
Lmfao

**Comment by srikarnarra1603** [comment]
lmaoo

**Comment by ImNimboss** [comment]
@pogrammar @SrikarNarra16 @EmreTech never mind, that was a fork of a Pycord bot. not actually theirs

**Reply by pogrammar** [reply]
fork of the pycord bot????

**Reply by pogrammar** [reply]
@ImNimboss

**Reply by EmmmaTech** [reply]
What? What do you mean by that?

**Reply by EmmmaTech** [reply]
Oh I see nevermind

**Comment by pogrammar** [comment]
@alphxn 

I have updated the Readme.md

If there are any other things needed, edit it and raise a pull request !
